src/target/riscv/riscv-013.c

```diff
@@ -2460,7 +2460,7 @@ static int read_memory_progbuf_inner(struct target *target, target_addr_t addres
  * Read the requested memory, silently handling memory access errors.
  */
 static int read_memory_progbuf(struct target *target, target_addr_t address,
-		uint32_t size, uint32_t count, uint8_t *buffer)
+		uint32_t size, uint32_t count, uint8_t *buffer, unsigned progbufsize)
```

Comment:
There's no need to pass progbufsize in here. You can access it through target using the RISCV013_INFO macro. See e.g. the start of read_memory().

Reply:
Done